SEARCH.md

```diff
@@ -43,23 +43,23 @@ The index is described with the following JSON:
 | description   | Short document description, used in help center results under the title text                                                                             |
 | content       | Main document content                                                                                                                                    |
 | source        | Document source, currently either `devportal` or `helpcenter`                                                                                            |
-| sort_priority | Document sort priority in search results, smaller priority documents are shown first and same priority documents are sorted by Elasticsearch query score |
+| sort_priority | Document sort priority in search results, smaller priority documents are shown first and same priority documents are sorted by OpenSearch query score |
 
 In addition to these `url` field should be provided with every document but it is not indexed or used in search queries. `url` is used in search result `href`.
 
 ## Search query
 
-The Elasticsearch index is used through the Netlify function in [netlify/functions/search/search.js](netlify/functions/search/search.js). The exact query can be found in the file but the idea is:
+The OpenSearch index is used through the Netlify function in [netlify/functions/search/search.js](netlify/functions/search/search.js). The exact query can be found in the file but the idea is:
 
-- use OR-style query matching `title`, `description` and `content` (with ES `match_phrase_prefix`)
+- use OR-style query matching `title`, `description` and `content` (with OS `match_phrase_prefix`)
 - give higher value to `title`
 
 # Creating the index
 
 The index is created with [scripts/create_index.py](scripts/create_index.py). You can run it with
 
 ```
-make create-index ES_URL=https://es.url/here
```

Comment:
There are several more mentions of https://es.url/here, better to replace all of them and be consistent

Reply:
I could only find two in the docstrings. https://github.com/aiven/devportal/pull/611/commits/40b7f2da30c48d853a97691fbca20fd1e165ab67